packages/@okta/i18n/src/properties/login.properties

```diff
@@ -926,10 +926,10 @@ oie.email.mfa.title = Verify with your email
 oie.email.verify.primaryButton = Send me an email
 # {0} is a user's email address
 oie.email.verify.subtitleWithEmailAddress = Verify with an email link or enter a code sent to <$1>{0}</$1>.
-oie.email.verify.subtitleWithoutEmailAddress = Verify with an email link or enter a code sent to <$1>your email</$1>.
+oie.email.verify.subtitleWithoutEmailAddress = Verify with an email link or enter a code sent to your email.
 # {0} is a user's email address
 oie.email.verify.alternate.magicLinkToEmailAddress = An email magic link was sent to <$1>{0}</$1>. 
-oie.email.verify.alternate.magicLinkToYourEmail = An email magic link was sent to <$1>your email</$1>. 
+oie.email.verify.alternate.magicLinkToYourEmail = An email magic link was sent to your email. 
```

Comment:
Please do not modify existing keys, as it'll produce L10N errors for non-English translations.

Instead, you will need to create a new key, and remove the stale key once translations are provided for the new key.

[More information here](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/741293778/Requesting+and+Receiving+Translations+i18n+and+l10n#Editing-strings) about this process.

Reply:
@jmelberg-okta L10N errors will occur if we add new arguments and not while removing right? In this case we will see <$1> tag for non-english until translations is complete. we are targeting to merge this PR following the i18n release cadence. we were having a discussion about this here https://okta.slack.com/archives/CAZH2MWEL/p1641583735007400.

do you still think we need to have new key?